hs/WebParsing/TimeTableParser.hs

```diff
@@ -14,17 +14,17 @@ import Database.JsonParser
 import WebParsing.HtmlTable
 import WebParsing.ParsingHelp
 import WebParsing.TimeConverter
-import Config (dbStr)
+import Config (databasePath)
 
 -- | used as an intermediate container while extracting lecture and tutorial information
 -- from the table. Is is later converted into lecture or tutorial records by examinig the
 -- first letter of the section.
 data CourseSlot =
-  CourseSlot {
-              slotSection :: T.Text,
-              slotTime_str :: T.Text,
-              slotInstructor :: T.Text
-              } deriving Show
+    CourseSlot {
+                slotSection :: T.Text,
```

Comment:
I'm not a fan of this indentation style - I think we should switch to a 4-space indent for containers whose first element starts on a new line. What do you think?